data/json/items/ranged/pneumatic.json

```diff
@@ -71,6 +71,45 @@
       [ "underbarrel mount", 1 ]
     ]
   },
+  {
+    "id": "bbpr",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "symbol": "(",
+    "color": "light_gray",
+    "name": "big bore pneumatic rifle",
+    "description": "A big bore pneumatic rifle.  The integrated pumping mechanism can be used to refill the integral air reservoir before firing.",
+    "price": 240000,
+    "material": [ "steel", "wood" ],
+    "skill": "rifle",
+    "ammo": "50pellet",
+    "weight": 4211,
+    "volume": 11,
+    "use_action": "HAND_PUMP_INTEGRAL",
+    "bashing": 8,
+    "to_hit": -1,
+    "ranged_damage": -5,
+    "range": 30,
+    "dispersion": 95,
+    "durability": 7,
+    "looks_like": "m107a1",
+    "reload": 150,
+    "loudness": 18,
+    "barrel_length": 5,
+    "compressed_air_used": 100,
+    "compressed_air_reservoir": 5000,
+    "magazine_well": 1,
+    "magazines": [ [ "50pellet", [ "bbprmag" ] ] ],
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "sights", 1 ],
+      [ "sling", 1 ],
+      [ "stock", 1 ],
```

Comment:
Why are these not mount slots(minus the sling) as well?

Reply:
What? I don't understand the question.

Reply:
Why grip, sights, and stock modslots you can directly add mods to rather than the mounts you have to add first, like you have for the rail slot?

Reply:
> Why are grip, sights, and stock modslots you can directly add mods to rather than the mounts you have to add first, like you have for the rail slot?

Dunno. I guess because it's DIY so I just used the tihar as the template, minus the underbarrel mount as that is where the pumping mechanism is presumably. What set of mods do you recommend? Did you have an existing (in CDDA) weapon in mind?

Reply:
Haven't heard in six days so I assume you are OK with it as is.